WooCommerce/UITestsFoundation/Screens/Login/GetStartedScreen.swift

```diff
@@ -1,4 +1,3 @@
-import UITestsFoundation
```

Comment:
This import is no longer needed because the file is now part of the framework.

Conversely, some UI test target files will now have to `import UITestsFoundation` in order to access the screens they require.